text/utilities/0000-syft-buildpack.md

```diff
@@ -0,0 +1,45 @@
+# Syft Buildpack
+
+## Summary
+
+Introduce a Syft buildpack. This buildpack will provide the `syft` binary should any buildpacks want to use it. The `syft` CLI is a tool and library for generating a Software Bill of Materials from container images and filesystems.
+
+The Syft buildpack would be maintained by the existing [Utilities team](https://github.com/orgs/paketo-buildpacks/teams/utilities). It uses libpak & libcnb.
+
+## Motivation
+
+It can be used by buildpacks to scan and generate both CycloneDX and SPDX listings of layers for compliance with [Buildpacks RFC #95](https://github.com/buildpacks/rfcs/blob/main/text/0095-sbom.md).
+
+## Detailed Explanation
+
+1. A Syft buildpack has been developed. It will be contributed to Paketo.
```

Comment:
Out of curiosity, have you already created the buildpack? Can we try it out?

Reply:
yes & yes. I'll DM you about it.